src/optionstr.c

```diff
@@ -2839,6 +2840,57 @@ did_set_imactivatekey(optset_T *args UNUSED)
 }
 #endif
 
+/*
+ * The 'isexpand' option is changed.
+ */
+    char *
+did_set_isexpand(optset_T *args)
+{
+    char_u  *ise = p_ise;
+    char_u  *p;
+    int	    last_was_comma = FALSE;
+
+    if (args->os_flags & OPT_LOCAL)
+	ise = curbuf->b_p_ise;
+
+    for (p = ise; *p != NUL;)
+    {
+	if (*p == '\\' && p[1] == ',')
+	{
+	    p += 2;
+	    last_was_comma = FALSE;
+	    continue;
+	}
+
+	if (*p == ',' && p[1] == ',')
+	{
+	    if (p[2] != ',' && p[2] != NUL)
+		return e_invalid_argument;
+	    last_was_comma = FALSE;
+	    p++;
+	    continue;
+	}
```

Comment:
Is this another way to specify a literal comma? I don't there there needs to be two ways to do that, especially in this case the comma can only be at the end of a trigger string.